### PR TITLE
Added the first example of style check in H2O-3.

### PR DESCRIPTION
**Pull request by vpatryshev**
We can introduce Google's style guide gradually, case by case, and fix it.
Chunk.getBytes() renamed to Chunk.bytes() (you can check out Josh Bloch's opinion on the issue of "getters").

### CODE REVIEW

**Comment by vpatryshev** [comment]
@h2o-ops please test!

File: gradle/config/checkstyle/h2o_checks.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    Authors: Max Vetrenko, Ruslan Diachenko, Roman Ivanov.
+    
+    This file is specific for H2O.ai. We plan to grow it gradually.
+ -->
+
+<module name = "Checker">
+  <module name="TreeWalker">
+    <module name="Regexp">
+        <property name="format" value="[^)]\.getBytes\("/>
```
Comment (mmalohlava) [inline gradle/config/checkstyle/h2o_checks.xml]: Could we find a better solution? A simple rule is causing too many unnecessary changes in code...

Comment (mmalohlava) [inline gradle/config/checkstyle/h2o_checks.xml]: Maybe: `"[^"]".getBytes(` could be enough.

Comment (mmalohlava) [inline gradle/config/checkstyle/h2o_checks.xml]: Also customs message would be nice:

```
<module name="MemberName">
    <property name="format" value="^m[a-zA-Z0-9]*$"/>
    <message key="name.invalidPattern"
             value="Member ''{0}'' must start with a lowercase ''m'' (checked pattern ''{1}'')."
             />
</module>```

Comment (vpatryshev) [inline gradle/config/checkstyle/h2o_checks.xml]: I'll add custom message; regarding using only string literals, it probably won't work; there's a lot of cases where a string is in a variable. I don't see a way to limit to specific data types - it's all on a macro level.

Comment (mmalohlava) [inline gradle/config/checkstyle/h2o_checks.xml]: I know, just for me seems better to have slightly less strict check style for now than changing API. Furthermore, every person who introduce `getBytes` method will be forced to change naming. We need to figure out slightly better check...

**Comment by vpatryshev** [comment]
@h2o-ops please test!

**Comment by mmalohlava** [comment]
I still think this is good change. Even a small change is an improvement...
